src/dotnet-interactive/Connection/KqlDiscoverabilityKernel.cs

```diff
@@ -25,7 +25,7 @@ public KqlDiscoverabilityKernel() : base(DefaultKernelName)
             "MsKqlKernel"
         };
         KernelInfo.LanguageName = "KQL";
-        KernelInfo.Description = $"""
+        KernelInfo.Description = """
                             Query a Kusto database
```

Comment:
minor nit: This could be a simple string.

Reply:
I find this syntax requires less cognitive overhead when reading the code. A lot of these user-facing strings contain characters that require escaping, so the raw string syntax makes it clear what the user will actually see, even if in some specific cases like this, there's no escaping present.